src/components/speaking-event/speaking-event.twig

```diff
@@ -0,0 +1,37 @@
+{% set _note %}{% block note %}{{ note }}{% endblock %}{% endset %}
+
+<div class="c-speaking-event{% if class %} {{class}}{% endif %}">
+  {% embed '@cloudfour/objects/media/media.twig' with {
+    reverse_markup: true,
+    datetime: datetime,
+    note: _note,
+    event: event,
+    location: location,
+    speakers: speakers,
+    href: href
+  } only %}
+    {% block object %}
+      {% include '@cloudfour/components/calendar-date/calendar-date.twig' with {
+        datetime: datetime,
+        note: note
+      } only %}
+    {% endblock %}
+    {% block content %}
+      <h3>
```

Comment:
Should this tag name be a property we can override in case a document calls for a specific semantic hierarchy?

Reply:
Oh, yes, great call!

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1488/commits/804e2af9e4707c4bc3d97fe72cb1fbd461abb026